frontends/web/src/app.tsx

```diff
@@ -210,7 +210,12 @@ export const App = () => {
             />
             <RouterWatcher />
           </div>
-          {showBottomNavigation && <BottomNavigation />}
+          {showBottomNavigation && (
+            <BottomNavigation
+              onlyHasOneAccount={activeAccounts.length === 1}
+              accountCode={activeAccounts[0]?.code}
```

Comment:
an alternative could be to just pass activeAccounts, then do the check inside BottomNavigation component